06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,15 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+
+####Link to pull request from branch setting-up-repository to development branch
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/pull/2)
+
+####Link to GitHub Repo
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/tree/chore/1/setting-up-repository)
```

Comment:
There's something wrong with your `ReadMe`.

The Markdown format is showing without the appropriate representation.